src/style.css

```diff
@@ -1,3 +1,77 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  --header-color: #00acdc;
```

Comment:
Variable name is not perfect - please be more clear what you use the color for

Better choices : `header-color-active` `header-font-active` `header-active`